bug(integrations): restore marketplace region label contract

## Problem

The broad Go gate currently fails in `internal/app` because the integrations provider configuration template no longer satisfies the provider input label contract.

## Route / surface

- Integrations provider configuration UI
- Contract test: `internal/app/ui_template_contract_test.go`

## Expected behavior

The provider config template includes a visible label bound to `provider-marketplace`:

```text
<Label htmlFor='provider-marketplace'>Marketplace / Region</Label>
id='provider-marketplace'
```

## Actual behavior

`go test ./... -count=1` fails with:

```text
--- FAIL: TestIntegrationsProviderConfigInputsHaveLabels
    ui_template_contract_test.go:588: integrations provider input label contract missing token: <Label htmlFor='provider-marketplace'>Marketplace / Region</Label>
FAIL github.com/collectors-tech/cabinet/internal/app
```

## Repro

```powershell
go test ./... -count=1
```

## Evidence

- Log: `.work-agent/logs/issue-842-seller-ops/2026-05-24T0530Z-go-test-all.log`
- Observed while validating PR #922 for #842.
- PR #922 changes only eBay seller operations/spec/traceability files, so this appears outside that diff.

## Scenario class

Regression / accessibility-label contract / PR merge gate blocker.

## Requirement / traceability

SPEC_GAP until mapped to the integrations provider settings/accessibility requirement.

## Proposed verification target

- `go test ./internal/app -run TestIntegrationsProviderConfigInputsHaveLabels -count=1`
- `go test ./... -count=1` before PR merge when practical.

--- a/ui.web/src/features/apps/index.tsx
+++ b/ui.web/src/features/apps/index.tsx
@@ -1539,9 +1539,7 @@ export function Apps({
                     />
                   </div>
                   <div className='space-y-2'>
-                    <Label htmlFor='provider-marketplace'>
-                      Marketplace / Region
-                    </Label>
+                    <Label htmlFor='provider-marketplace'>Marketplace / Region</Label>
                     <Input
                       id='provider-marketplace'
                       placeholder='Marketplace / Region'
@@ -1555,9 +1553,7 @@ export function Apps({
                     />
                   </div>
                   <div className='space-y-2'>
-                    <Label htmlFor='provider-items-per-page'>
-                      Items per page
-                    </Label>
+                    <Label htmlFor='provider-items-per-page'>Items per page</Label>
                     <Input
                       id='provider-items-per-page'
                       type='number'
@@ -1594,9 +1590,7 @@ export function Apps({
                         </div>
                       ) : (
                         <div className='space-y-2'>
-                          <Label htmlFor='provider-token'>
-                            New token / API key
-                          </Label>
+                          <Label htmlFor='provider-token'>New token / API key</Label>
                           <Input
                             id='provider-token'
                             type='password'

